packages/vulcan-core/lib/modules/default_resolvers.js

```diff
@@ -133,7 +133,7 @@ export function getDefaultResolvers(options) {
           } else {
             throwError({
               id: 'app.missing_document',
-              data: { documentId, selector },
+              data: { documentId, selector, collectionName: collection.collectionName },
```

Comment:
Is this related to the rest of changes in this particular commit?

Reply:
It adds slightly better information to error logging, which I wound up using to diagnose one of the things that's fixed here. But it isn't itself a bug.